typespec-extension/src/code-model-builder.ts

```diff
@@ -1523,7 +1516,9 @@ export class CodeModelBuilder {
       for (const response of resp.responses.values()) {
         if (response.headers) {
           for (const [key, header] of Object.entries(response.headers)) {
-            const schema = this.processSchema(header, key);
+            // TODO: change to sdk type
```

Comment:
Is this TODO still needed?

Reply:
Thanks,  I already deleted locally, will commit and push.